src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributes/relationship.svelte

```diff
@@ -39,8 +39,6 @@
             }
         }
 
-        documentList = await getDocuments();
-
```

Comment:
Was the removal intentional? Where do we do this now?

Reply:
> Was the removal intentional? Where do we do this now?

yep, we do this now -
`$: getDocuments(search).then((res) => (documentList = res));` on line `108`.